worker/uniter/runner/jujuc/secret-add.go

```diff
@@ -117,6 +122,25 @@ func (c *secretUpsertCommand) Init(args []string) error {
 		return errors.NotValidf("secret owner %q", c.owner)
 	}
 
+	// Process keys where content is to come from a file.
```

Comment:
Have we already supported #base64?
I cannot find a code to handle this yet.
It'd be good to move these lines to a separate function and put to #base64 handlers together.

Reply:
#base64 is supported in the core secrets package. We can't really more this #file code there because the core secrets package expects to deal with just the content, not where the content comes from

Reply:
Actually, in core we do have the code to load an entire file, so I moved the #file processing there was well.